hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContract.php

```diff
@@ -522,4 +522,34 @@ public static function getActiveContracts($startDate = null, $endDate = null)
 
     return $contracts;
   }
+
+  /**
+   * Return the current contract for the contact if exist.
+   * @param int $cuid contact ID
+   * @return array|null return an associative array contain current contract details
+   *         or null if there is no current contract for the desired contact
+   */
+  public static function getCurrentContract($cuid)  {
+    // get current absence period
```

Comment:
I think you can get rid of this comment. The method documentation + the method name are enough